recipes/python-rocksdb/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+c_compiler: gcc
```

Comment:
This file in general shouldn't be needed, it looks like a manual copy of what smithy is generating, please remove it.

Reply:
Removed this file